static/js/zamboni/buttons.js

```diff
@@ -179,16 +178,28 @@ var installButton = function() {
         };
     });
 
+    var showDownloadAnyway = function($button) {
+        var $installShell = $button.parents('.install-shell');
+        var $downloadAnyway = $installShell.find('.download-anyway');
+        if ($downloadAnyway.length) {
+            var href = $button.filter(':visible').attr('href');
+            var pos = href.lastIndexOf('/');
+            href = escape_(href.slice(0, pos) + '/type:attachment/' + href.slice(pos + 1));
```

Comment:
This might not work, our regular expression in `versions.urls` expects `type` to come before `platform` and this does the reverse. Unless this is a `file/:id/` link?